### PR TITLE
Remove pooling in LocalParallelPipeline [Resolves #122]

### LINKED ISSUE #122: Disable pooling in LocalParallelPipeline

### PR DESCRIPTION
**Pull request by thcrock**

### CODE REVIEW

File: triage/pipelines/local_parallel.py [hunk 1]
```diff
@@ -211,13 +212,15 @@ def insert_into_table(
 ):
     try:
         logging.info('Beginning insert batch')
-        db_engine = create_engine(db_connection_string)
+        db_engine = create_engine(db_connection_string, poolclass=NullPool)
```
Comment (nanounanue) [inline triage/pipelines/local_parallel.py]: Question: What is the purpose of this change?

**Comment by thcrock** [comment]
I closed the issue that this tried to address ( #122 ), so closing this as well.
